src/main/asciidoc/types/index.adoc

```diff
@@ -61,16 +61,15 @@ The Neo4j driver maps Neo4j types to native language types as follows:
 | Neo4j        | Java
 | Null         | null
 | Boolean      | java.lang.Boolean
-| Integer      | java.lang.Integer
+| Integer      | java.lang.Long
 | Float        | java.lang.Double
 | String       | java.lang.String
 | List         | java.util.List<T>
-| Map          | java.util.Map<String, Value> (*)
+| Map          | java.util.Map<TKey, TValue>
```

Comment:
Thanks for the PR @zhenlineo! This line for the Java driver, right? Is it really `Map<TKey, TValue>` in Java? That sounds like the .NET map.